### DISCUSSION TITLE
How Will CVE-2024-23832 "Work" for Non-Patched vs. Patched Instances?

### DISCUSSION THREAD

**Discussion by Shanesan** [mastodon/mastodon#29134] [Q&A] [answered]
Quick question on [CVE-2024-23832](https://github.com/mastodon/mastodon/security/advisories/GHSA-3fjr-858r-92rw)...

So say you have an unpatched Mastodon instance and a patched Mastodon instance. Will the patched instances just drop posts/calls/etc from unpatched instances? Or will a patched instance potentially accept all posts from unpatched instances, including impersonation from those instances?

**Comment by Gargron** [comment, marked as answer]
If your server is patched it won't have an issue with patched/unpatched servers. The problem is on the receiving end only.
